doc/api/util.md

```diff
@@ -10,6 +10,42 @@ module developers as well. It can be accessed using:
 const util = require('util');
 ```
 
+## util.callbackify(original)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `original` {Function} An async function, or a Promise factory
+* Returns: {Function} a "callback style" function
+
+Takes an async function and returns a function following the common Node.js
+callback style, i.e. taking a `(err, val) => ...` callback as the last argument.
+
+For example:
+
+```js
+const util = require('util');
+async function fn() {
+  return await Promise.resolve('hello world');
+}
+const cbFn = util.callbackify(fn);
+
+cbFn((err, ret) => {
+  if (err) throw err;
+  console.log(ret);
```

Comment:
It might be good to add a comment somewhere indicating that `err` will be the rejection reason if the Promise resolves, and `ret` will be the value of the Promise fulfills.